lib/Service/FeedServiceV2.php

```diff
@@ -211,7 +204,21 @@ public function create(
             list($feed, $items) = $this->feedFetcher->fetch($feedUrl, $full_text, $user, $password);
         } catch (ReadErrorException $ex) {
             $this->logger->debug($ex->getMessage());
-            throw new ServiceNotFoundException($ex->getMessage());
+            # the url did not lead to a valid feed, try autodiscover
+            if ($full_discover) {
```

Comment:
Did this on mobile so the formatting is a pain, but let's do it like this.

```suggestion
            if ($full_discover === false) {
throw new ServiceNotFoundException($ex->getMessage());
}
```

Reply:
Thanks, I will adjust the code soon :)